doc/source/doc-style/doc-configuration.rst

```diff
@@ -0,0 +1,68 @@
+<!-- vale off -->
+Required Sphinx configuration
+=============================
+<!-- vale on -->
```

Comment:
Why are you using vale off/on ? I'd prefer that you add "Sphinx" to the doc/styles/Vocab/accept.txt file. That is the place where "ignored" or "admitted" words are located

Reply:
Oh! I tried to add it to the `accept.txt` but Vale then complained about `sphinx` without capital case. I think if I add both it should work, let me check.

Reply:
Yep, that should work :)